test/examples/ucp_client_server.c

```diff
@@ -296,7 +296,9 @@ static void usage()
     fprintf(stderr, " -a Set IP address of the server "
                     "(required for client and should not be specified "
                     "for the server)\n");
-    fprintf(stderr, " -p Set alternative server port (default:13337)\n");
+    fprintf(stderr, " -p Set alternative server port (default:13337)\n"
+                    "    Setting 0 as the port, will generate a random port for "
```

Comment:
"-p <port>    Port number to listen/connect to. 0 on the server side menas select random port and print it"